### DISCUSSION TITLE
Welcome to lage Discussions!

### DISCUSSION THREAD

**Discussion by kenotron** [microsoft/lage#117] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sunilsurana** [comment]
Can we control max number of processes spawned by lage? I don't see any config option for doing that.

**Reply by kenotron** [reply]
It is called `--concurrency`
